corehq/apps/app_manager/tests/test_translations.py

```diff
@@ -0,0 +1,19 @@
+# coding=utf8
+from lxml import etree
+from django.test import SimpleTestCase
+from corehq.apps.app_manager.translations import escape_output_value
+
+
+class AppManagerTranslationsTest(SimpleTestCase):
+
+    def test_escape_output_value(self):
+        test_cases = [
+            ('hello', '<value>hello</value>'),
+            (u'क्लिक', '<value>&#2325;&#2381;&#2354;&#2367;&#2325;</value>'),
+            ('&#39', '<value>&amp;#39</value>'),
+            ('Here is a ref <output value="/data/no_media" /> with some trailing text and bad < xml.',
+             '<value>Here is a ref <output value="/data/no_media"/> with some trailing text and bad &lt;xml.</value>')
```

Comment:
the whitespace after the `&lt;` gets swallowed. wasn't sure if that was intentional or not